autoload/neomake/makers/python.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#python#EnabledMakers()
-    return ['python', 'pep8', 'pyflakes', 'pylint']
+    return ['python', 'pep8', 'pyflakes', 'pylint', 'flake8']
```

Comment:
Since flake8 runs pep8 and pyflakes, it seems that if all three are in the list of enabled makers we're going to get some redundant runs. It's more complicated, but I think it would be better if we did something like this:

```
if exists('s:python_makers')
    return s:python_makers
endif
let makers = ['python']
if neomake#utils#Exists('flake8')
    add(makers, 'flake8')
else
    extend(makers, ['pep8', 'pyflakes'])
endif
add(makers, 'pylint')  " Last because it is the slowest
let s:python_makers = makers
return makers
```